include/revng/Model/Binary.h

```diff
@@ -128,6 +44,9 @@ class model::Binary : public model::generated::Binary {
 public:
   using generated::Binary::Binary;
 
+private:
+  static constexpr uint64_t SupportedVersion = 1;
+
```

Comment:
This should be public and automatically injected from `tuple-tree-generator`

Reply:
Done.

Reply:
Ok.
@khaledismaeel  if you address a comment, please make sure to resolve the conversation.